### PR TITLE
pkg5: PEP8 compliancy and documentation changes

### PR DESCRIPTION
**Pull request by dagwieers**
##### SUMMARY
This PR includes:
- PEP8 compliancy changes
- Documentation changes

##### ISSUE TYPE
 - Feature Pull Request
 - Docs Pull Request

##### COMPONENT NAME
pkg5

##### ANSIBLE VERSION
v2.5

### CODE REVIEW

File: lib/ansible/modules/packaging/os/pkg5.py [hunk 1]
```diff
@@ -144,16 +126,7 @@ def ensure(module, state, packages, params):
...
-            [
-                'pkg', behaviour[state]['subcommand']
-            ]
-            + dry_run
-            + accept_licenses
-            + [
-                '-q', '--'
-            ] + to_modify
-        )
+        rc, out, err = module.run_command(['pkg', behaviour[state]['subcommand']] + dry_run + accept_licenses + ['-q', '--'] + to_modify)
```
Comment (mavit) [inline lib/ansible/modules/packaging/os/pkg5.py:129]: I find these kinds of long line less readable, harder to work with, and less helpful with `git blame`, but I realise that this is all subjective, so I'm going to try to resist arguing about the colour of this particular bikeshed.

**Review by mavit — APPROVED** [review]
@ansibot: shipit

File: lib/ansible/modules/packaging/os/pkg5.py (hunk 1 above)
Comment (dagwieers) [reply lib/ansible/modules/packaging/os/pkg5.py:129]: I don't mind doing it differently though. The original was not PEP8 compliant, but there is a PEP8 compliant multi-line solution. Our PEP8 implementation also supported 160 wide lines, so I opted for that.

Comment (dagwieers) [reply lib/ansible/modules/packaging/os/pkg5.py:129]: It's already merged, feel free to implement your preference

**Pull request merged**